src/kokkos_ext/ArborX_DetailsKokkosExtMinMaxOperations.hpp

```diff
@@ -73,6 +76,80 @@ KOKKOS_INLINE_FUNCTION constexpr T min(std::initializer_list<T> ilist)
   return result;
 }
 
+template <typename ExecutionSpace, typename ViewType>
+std::pair<typename ViewType::non_const_value_type,
+          typename ViewType::non_const_value_type>
+minmax_element(ExecutionSpace &&space, ViewType const &v)
```

Comment:
Is there any good reason to use a forwarding reference for `ExecutionSpace` (apart from mimicking `ExecutionPolicy` for the `std` algorithms)?

Reply:
I don't think so, and it's annoying (as `is_accessible_from` or `Kokkos::is_execution_space` won't work without decaying). I will change it to const-ref, same as it's done for StdAlgorithms in Kokkos.